js/apps/admin-ui/src/clients/registration/ClientRegistrationList.tsx

```diff
@@ -104,7 +104,9 @@ export const ClientRegistrationList = ({
       <DeleteConfirm />
       <KeycloakDataTable
         ariaLabelKey="clientRegistration"
-        searchPlaceholderKey="searchClientRegistration"
+        searchPlaceholderKey={t("searchClientRegistration", {
+          policyType: subTab || "anonymous",
```

Comment:
We'll have to localize `subTab` and `"anonymous"` as well if they are to be displayed in different languages.

Reply:
I see, I'll keep it simple then.